pkg/api/types.go

```diff
@@ -334,6 +467,8 @@ type Capabilities struct {
 type ResourceRequirements struct {
 	// Limits describes the maximum amount of compute resources required.
 	Limits ResourceList `json:"limits,omitempty"`
+	// Requests describes the minimum amount of compute resources required.
+	Requests ResourceList `json:"requests,omitempty"`
```

Comment:
@bgrant0607 I'd rather globally rename Limits -> Requests than add both.  I am worried about the implied semantics of Request != Limit.